packages/appstore/src/types/common.types.ts

```diff
@@ -171,7 +171,7 @@ export interface AccountListDetail {
     title: string;
 }
 
-export type TWalletsMigrationStatus = 'ineligible' | 'eligible' | 'in_progress' | 'migrated' | 'failed';
+export type TWalletsMigrationStatus = 'ineligible' | 'eligible' | 'in_progress' | 'done' | 'failed';
```

Comment:
Changed this based on this [thread](https://deriv-group.slack.com/archives/C0D4F6W7N/p1681892955449359)